src/Catalog/PackageDeprecationItem.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Versioning;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public class PackageDeprecationItem
+    {
+        public PackageDeprecationItem(
```

Comment:
Can't this be the same entity as what gallery uses?

Reply:
Because we aren't using EF, it seems very odd to me to take a dependency on `NuGet.Services.Entities` only for the purpose of converting the output of a SQL query into an EF model before we serialize it into something else. If it was EF I'd agree.

Reply:
Okay, sounds good.